test/kubernetes/e2e/features/client_tls/suite.go

```diff
@@ -36,139 +36,91 @@ func NewTestingSuite(ctx context.Context, testInst *e2e.TestInstallation) suite.
 }
 
 func (s *clientTlsTestingSuite) SetupSuite() {
```

Comment:
should we make sure the pods are running with `s.testInstallation.Assertions.EventuallyPodsRunning`